### ISSUE TITLE
Cannot connect to the server

### ISSUE THREAD

**Issue by rfwankenobi** [NickAkincilar/QlikSense-Task-Failure-Email-Alerts#3]
Hi there,

When I try to run the task I get this error:

Time, LogLevel, Message
2017_02_25_11_07_21,Information,"Proxy: http://localhost/"
2017_02_25_11_07_21,Information,"Looking for Failed Tasks..."
2017_02_25_11_07_21,Error,"Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: <. Path '', line 0, position 0.
   at Newtonsoft.Json.JsonTextReader.ParseValue()
   at Newtonsoft.Json.JsonTextReader.Read()
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.ReadForType(JsonReader reader, JsonContract contract, Boolean hasConverter)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)
   at QlikSenseJSONObjects.QlikSenseJSONHelper.GetTaskByStatus(QSTaskStatus status)
   at QlikSenseEmailAdmin.Program.Main(String[] args)
"
2017_02_25_11_07_21,Information,"Returning Errorlevel 8"

What port needs to be open so that your tool works? I successfully used your tool in the past, but in my new install of  Sense 3.2.1 it doesn't work. When I click on "Test connection" it always fails, even after I took the firewall down...

The tool is trying to connect to http://localhost/

Best,

Rodolfo

**Comment by NickAkincilar** [comment]
Please try installing version 1.4a if not already . It outputs more detail in the error log file which I will need a copy to diagnose.

**Comment by rfwankenobi** [comment]
No luck. I tried the old version also, and I couldn't get it to "Test Connection" successfully. I tried both HTTP and HTTPS urls, both public and local, and no log was generated and no other message was received.

I saw this same issue before: "Unexpected character encountered while parsing value: <. Path ''" This was due to my install not using the Windows's proxy by default, but using the "Forms" login instead. I tried forcing the tool to use .../windows/ too (which is my proxy prefix), but that didn't work either.
